server/models/action-inputs-model.js

```diff
@@ -1,15 +1,15 @@
-function getTitle (actionId) {
+function getTitle (actionId, parcelRef) {
   return actionId === 'FG1' ? 'Enter a fence length in metres' : 'Enter the area in hectares'
 }
 
-function getHint (actionId) {
-  return actionId === 'FG1' ? 'How long is the fence you want to claim on in metres?' : 'What is the area you wish to claim on in hectares?'
+function getHint (actionId, parcelRef) {
+  return actionId === 'FG1' ? 'How long is the fence you want to claim on in metres for ' + parcelRef + '?' : 'What is the area you wish to claim on in hectares for ' + parcelRef + '?'
```

Comment:
you could use a string template rather than concatenation here, i.e.
```
`How long is the fence you want to claim on in metres for ${parcelRef}?`
```

Reply:
That would be the sensible thing to do. Not sure why I didn't think about that.